src/iddb.rho

```diff
@@ -74,7 +74,7 @@ in {
                                             [target, ...rest] => {
                                                 if (keys.contains(target)) {
                                                     for (@record <<- @{[*self, table_name, target]}) {
-                                                        loop!(rest, acc.put(target, record))
+                                                        loop!(rest, acc.set(target, record))
```

Comment:
ugh. I clearly never tested this.

Bonus points for updating the "unit test" stuff below to exercise this code.

Reply:
Kudos for having only one error without testing it. :)

Reply:
@dckc Add some basic tests https://github.com/rchain/rchat/pull/9/commits/32cfd17dfe1485d1cc8257a9ed75dfc652a0ecde.